hack/community/generate.sh

```diff
@@ -45,67 +53,33 @@ replace() {
     sed -i -e "s/"preview,stable"/$MATURITY/" -e "s/"stable"/$MATURITY/" "${CO_ANNOTATIONS}"
 }
 
-# copy the WMCO bundle and its contents to a new community folder. Rename files
-# as needed.
+# Copy the WMCO bundle and its contents to the output directory.
 generate_manifests() {
-  local OPERATOR_VERSION=$1
-  local BUNDLE_DIR=$2
-  local DESCRIPTION=$3
-  local QUAY_IMAGE=$4
-  local CO_OPERATOR_DIR="$(pwd)/operators/community-windows-machine-config-operator"
+  local BUNDLE_DIR=$1
+  local DESCRIPTION=$2
+  local OUTPUT_DIR=$3
 
   echo "Update operator manifests"
-  mkdir -p "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}"
-  cp -r "${BUNDLE_DIR}/manifests" "${BUNDLE_DIR}/metadata" "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}"
-  mv "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}/manifests/windows-machine-config-operator.clusterserviceversion.yaml" "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}/manifests/community-windows-machine-config-operator.${OPERATOR_VERSION}.clusterserviceversion.yaml"
-  local CO_CSV="${CO_OPERATOR_DIR}/${OPERATOR_VERSION}/manifests/community-windows-machine-config-operator.${OPERATOR_VERSION}.clusterserviceversion.yaml"
-  local CO_ANNOTATIONS="${CO_OPERATOR_DIR}/${OPERATOR_VERSION}/metadata/annotations.yaml"
-
-  replace "$CO_CSV" "$OPERATOR_VERSION" "$DESCRIPTION" "$QUAY_IMAGE" "$CO_ANNOTATIONS"
-}
-
-# Create a signed commit for the community-operators-prod repo
-commit() {
-  local OPERATOR_VERSION=$1
-  TITLE="operators community-windows-machine-config-operator.v$OPERATOR_VERSION"
+  cp -r "${BUNDLE_DIR}/manifests" "${BUNDLE_DIR}/metadata" "${OUTPUT_DIR}"
+  local CO_CSV="${OUTPUT_DIR}/manifests/windows-machine-config-operator.clusterserviceversion.yaml"
+  local CO_ANNOTATIONS="${OUTPUT_DIR}/metadata/annotations.yaml"
 
-  echo "commit changes"
-  git add --all
-  git commit -s -m "${TITLE}"
+  replace "$CO_CSV" "$DESCRIPTION" "$CO_ANNOTATIONS"
 }
 
-OPERATOR_VERSION="$1"
-COMMUNITY_VERSION="$2"
-QUAY_IMAGE="$3"
-CO_DIR="$4"
-OPERATOR_MANIFESTS="bundle/manifests"
-OPERATOR_METADATA="bundle/metadata"
-
-if [ -z "$QUAY_IMAGE" ]; then
-    echo "Must set QUAY_IMAGE to latest community tag"
-    exit 1
-fi
-
-if [ -z $CO_DIR ]; then
-  echo "CO_DIR not set"
-  exit 1
-fi
+WMCO_VERSION="$1"
+OUTPUT_DIR="$2"
 
-# Check if yq is installed
-if ! command -v which yq &> /dev/null; then
-  echo "Please install yq before running generate.sh. Directions to install yq are in the readme.md"
+if [ -z $OUTPUT_DIR ]; then
+  echo "OUTPUT_DIR not set"
   exit 1
 fi
 
-echo "Cutting community release for v${OPERATOR_VERSION}"
+get_co_version "$WMCO_VERSION"
```

Comment:
suggestion, instead of setting `COMMUNITY_VERSION` var within `get_co_version()`, have it return the version and make this line `COMMUNITY_VERSION = $(get_co_version "$WMCO_VERSION")`